app/models/court.rb

```diff
@@ -7,4 +7,15 @@ class Court < ApplicationRecord
     concrete: 1,
     tartan: 2
   }
+
+  def check_availability(date)
```

Comment:
czemu w modelu? no i nazwa metody slaba bo zwracasz tutaj boolean wiec jesli juz to na przyklad `available_at?`

Reply:
Gdzie w innym miejscu powinienem to zrobić?

Reply:
skoro uzywasz tylko w searchu to sugerowalbym tam

Reply:
Tylko będzie to też używane np. przy tworzeniu rezerwacji żeby sprawdzić czy można wtedy utworzyć